emission/storage/decorations/stats_queries.py

```diff
@@ -44,5 +46,23 @@ def store_stats_entry(user_id, metadata_key, name, ts, reading):
     "reading": reading
   }
   new_entry = ecwe.Entry.create_entry(user_id, metadata_key, data)
-  return esta.TimeSeries.get_time_series(user_id).insert(new_entry)
+  return esta.TimeSeries.get_non_user_time_series().insert(new_entry)
```

Comment:
@TeachMeTW you cannot change this line - it will break the existing uses of `store_stats_entry` (e.g. in `store_pipeline_time`).

You will need to check for the presence of `user_id` and call the appropriate method, or call `esta.TimeSeries.get_non_user_time_series().insert(new_entry)` directly here

https://github.com/e-mission/e-mission-server/pull/986/files#diff-0e4360245e864d2687db5f80e19494b632e5bfcc4fb27f75eca4672e75ee8091R63

Please document the pros and cons of that choice before making the change.